protos/repository.proto

```diff
@@ -36,27 +38,55 @@ message BentoMetadata {
     string docs = 3;
   }
 
-  BentoUri uri = 1;
-  string created_at = 2;
-  string name = 3;
-  string version = 4;
-  BentoServiceEnv env = 5;
-  repeated BentoArtifact artifacts = 6;
-  repeated BentoServiceApi apis = 7;
+  string name = 1;
```

Comment:
What's the different between name/version inside `BentoServiceMetadata` and `Bento`